gradle.properties

```diff
@@ -1,4 +1,4 @@
 GROUP=io.beekeeper
 VERSION=2.2.1
```

Comment:
We need to bump the version here as well, otherwise the publish will fail.

This is a major change, so I'd jump to 
```suggestion
VERSION=2.3.0
```
or even
```suggestion
VERSION=3.0.0
```

Reply:
I have bumped up the version to 2.3.0.